otter_welcome_buddy/cogs/english_club_reminder.py

```diff
@@ -0,0 +1,135 @@
+    """
+
+    def __init__(self, bot: Bot, messages_dependency: type[messages.Formatter]):
+        self.bot: Bot = bot
+        self.messages_formatter: type[messages.Formatter] = messages_dependency
+        self.scheduler: AsyncIOScheduler = AsyncIOScheduler()
+        self.__configure_scheduler()
+
+    @commands.group(
+        brief="Commands related to English Club!",
+        invoke_without_command=True,
+        pass_context=True,
+    )
+    @commands.has_role(OTTER_MODERATOR)
+    async def englishclub(self, ctx: Context) -> None:
+        """
+        English Club will setup an English session
+        """
+        await ctx.send_help(ctx.command)
+
+    @englishclub.command(brief="Start the cronjob for the messages")  # type: ignore
+    @commands.has_role(OTTER_MODERATOR)
+    async def start(self, _: Context) -> None:
+        """Command to interact with the bot and start cron"""
+        self.__configure_scheduler()
+
+    @englishclub.command(brief="Stop the cronjob for the messages")  # type: ignore
+    @commands.has_role(OTTER_MODERATOR)
+    async def stop(self, _: Context) -> None:
+        """Command to interact with the bot and stop cron"""
+        self.scheduler.remove_job("job_id")
+
+    @commands.command(brief="Schedule an English session")
+    @commands.has_role(OTTER_MODERATOR)
+    async def schedule_session(self, ctx: Context, *hour: str) -> None:
+        """Schedule the English Club session"""
+
+        # Regex pattern explanation:
+        # ^: Asserts the start of the string.
+        # (1[0-2]|0?[1-9]): Matches an hour with the 12-hour format.
+        # : Matches the colon that separates the hour and minute.
+        # ([0-5][0-9]): Matches exactly 2 digits for the minutes, from "00" to "59".
+        #  ?: Matches zero or one space, allowing for space between the minutes and AM/PM.
+        # (AM|PM): Matches either "AM" or "PM" (case-sensitive).
+        # $: Asserts the end of the string.
+        time_format_regex = r"^(1[0-2]|0?[1-9]):([0-5][0-9]) ?(AM|PM)$"
+        hour_str: str = " ".join(hour)
+
+        if re.match(time_format_regex, hour_str):
+            channel_id: int = ENGLISH_CHANNEL_ID
+            channel: DiscordChannelType | None = get_channel_by_id(self.bot, channel_id)
+            everyone = DiscordCommonRoles.EVERYONE.value
+
+            embed = discord.Embed(
+                colour=discord.Colour.green(),
+                title="English Club Session",
+                description=everyone + self.messages_formatter.english_club_message(hour_str),
+            )
+
+            embed.set_image(url=OTTER_HEADER_IMAGE)
+
+            if isinstance(channel, TextChannel):
+                await channel.send(embed=embed)
+        else:
+            await ctx.send(
+                "Invalid, Use the following format: HH:MM AM or HH:MM PM, e.g., '10:00 PM'.",
+            )
+
+    def message_non_english_club(self) -> str:
```

Comment:
As good practice, the functions that will be used only in this class are considered either private (just this class) or protected (this and any class that extend to this). This is a OOP thing and python doesn't really follow that, but as naming convention you declare a private function with two underscores before `def __foo()` and a protected function with one underscore `def _foo()`. You can read a bit more here:
https://www.tutorialsteacher.com/python/public-private-protected-modifiers
So, in this case, change this function to:
```suggestion
    def _message_non_english_club(self) -> str:
```
You can use double underscore but just one is enough.